start-alertmanager.sh

```diff
@@ -85,4 +86,8 @@ then
 fi
 
 AM_ADDRESS="$(docker inspect --format '{{ .NetworkSettings.IPAddress }}' $ALERTMANAGER_NAME):9093"
+if [ ! -z "$is_podman" ] && [ "$AM_ADDRESS" = ":9093" ]; then
+    HOST_IP=`hostname -I | awk '{print $1}'`
```

Comment:
Again, I don't understand the meaning of taking the first response in "hostname -I", which is a list of IP address whose order is undefined.